### DISCUSSION TITLE
🔊🔊🔊Please Tell Me Anything🔊🔊🔊

### DISCUSSION THREAD

**Discussion by AtropsCooper** [NAIST-SE/AttractingContributors#404] [General]
🤟  Here is a list of purely random. 

🔊 **DON'T** miss the chance to post **ANYTHING**!  🔊 
🔊 **COME** and **POST** right now! 🔊 

PRs will be merged and PRed back before February 🥺

My fork:
https://github.com/AtropsCooper/AttractingContributors-Random

⏬ And don't forget to **HIT the LIKE button**!!!

**Comment by XC1996114514** [comment]
❤❤❤

**Comment by aru1702** [comment]
ᗜ_ᗜ
See my repo >> https://github.com/aru1702/AttractingContributors-TouhouFumo <<

**Reply by AtropsCooper** [reply]
I have added koishi 😄

**Reply by AtropsCooper** [reply]
I'm glad that your favorite is also Koishi.
Maybe we can be friends 😘

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter

**Reply by AtropsCooper** [reply]
Thank you! 
I added my favorite!

**Comment by AtropsCooper** [comment]
Please post something randomly here, no limit.

**Comment by ani-hovhannisyan** [comment]
Hi, I have added 19th line in your project, can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git

**Reply by AtropsCooper** [reply]
Thank You! 🙇‍♂️ 
I have added my favorite to your list.
